src/resolver/resolve_path.zig

```diff
@@ -581,8 +716,9 @@ pub const Platform = enum {
         return switch (comptime platform) {
             .auto => (comptime platform.resolve()).isAbsolute(path),
             .posix => path.len > 0 and path[0] == '/',
-            .windows => std.fs.path.isAbsoluteWindows(path),
-            .loose => isAbsolute(.posix, path) or isAbsolute(.windows, path),
+            .windows,
+            .loose,
```

Comment:
windows applies to both?

Reply:
yes. you can think of isAbsoluteWindows as a superset of isAbsolutePosix